### PR TITLE
[Port Manager] Fix 400 Bad Request for Port List Calls with Excessive URL

### PR DESCRIPTION
**Pull request by xieus**
This PR resolves the 400 Bad Request issue on the PM side. The fix include three parts:
- Enlarge HTTP max header size to 16KB
- Add a few logs to API GW for ease of debugging in performance testing

### CODE REVIEW

**Comment by Gzure** [comment]
@xieus I test the large URL in my local environment, it causes by PM don't support large HTTP header, so I left some comment in [https://github.com/futurewei-cloud/alcor/issues/469](URL).

**Comment by xieus** [comment]
> @xieus I test the large URL in my local environment, it causes by PM don't support large HTTP header, so I left some comment in [https://github.com/futurewei-cloud/alcor/issues/469](URL).

Oh just saw your comment.. Yes you are right. After enabling the TRACE level, it showed PM returns 400 and further tracing back to PM found the error.  Somehow the error was not found in last round of investigation (maybe by mistake). The fix has been verified by a manual API test.

**Comment by xieus** [comment]
@Gzure Can you review this PR as well? Thanks.

File: services/api_gateway/pom.xml [hunk 1]
```diff
@@ -20,8 +20,8 @@
 
     <parent>
         <groupId>org.springframework.boot</groupId>
-        <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.1.6.RELEASE</version>
+            <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.3.5.RELEASE</version>
```
Comment (kevin-zhonghao) [inline services/api_gateway/pom.xml:24]: we could go back to the previous version of springframework

**Review by kevin-zhonghao — APPROVED** [review]

File: services/api_gateway/pom.xml (hunk 1 above)
Comment (xieus) [reply services/api_gateway/pom.xml]: Correct. Roll back to previous version of Spring Boot and Cloud GW.

**Pull request merged**